### PR TITLE
Power Up movement

### PR DESCRIPTION
**Pull request by DJSteins**
Adds movement to the power ups when they are thrown and places and rotates them based on the car's rotation and position.

### CODE REVIEW

File: ProjectSettings/GraphicsSettings.asset [hunk 1]
```diff
@@ -1,58 +1,60 @@
...
-    m_Shader: {fileID: 75, guid: 0000000000000000f000000000000000, type: 0}
-  m_LightHalo:
-    m_Mode: 1
-    m_Shader: {fileID: 105, guid: 0000000000000000f000000000000000, type: 0}
-  m_LensFlare:
-    m_Mode: 1
-    m_Shader: {fileID: 102, guid: 0000000000000000f000000000000000, type: 0}
-  m_AlwaysIncludedShaders:
-  - {fileID: 10753, guid: 0000000000000000f000000000000000, type: 0}
+%YAML 1.1
```
Comment (m4rtint) [inline ProjectSettings/GraphicsSettings.asset]: Wait do you know what you changed here?

File: ProjectSettings/EditorBuildSettings.asset [hunk 2]
```diff
@@ -17,4 +17,7 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/Scenes/Office.unity
     guid: e36ec9424def6ac49b3294ad1c9f2bb4
+  - enabled: 1
```
Comment (m4rtint) [inline ProjectSettings/EditorBuildSettings.asset]: How come you added the scene to the build?

File: ProjectSettings/GraphicsSettings.asset (hunk 1 above)
Comment (DJSteins) [reply ProjectSettings/GraphicsSettings.asset]: Nope :(

File: ProjectSettings/EditorBuildSettings.asset (hunk 2 above)
Comment (DJSteins) [reply ProjectSettings/EditorBuildSettings.asset]: No particular reason. Was just figuring some things out. I didn't even realize that changed this asset.

**Comment by mikeslgoh** [comment]
[Implement powerup movement](https://trello.com/c/P76z4ocY/334-implement-powerup-movement)

**Pull request merged**